### ISSUE TITLE
Provide wheel platform tag stats

### ISSUE THREAD

**Issue by lazka** [crflynn/pypistats.org#69]
Not sure if this data is available in big query etc, but I'd like to know which platform my wheels are downloaded for. For example 32bit vs 64bit vs arm64 on Windows.

**Comment by hugovk** [comment]
Yes, this sort of thing is in BigQuery, you can get it using the https://github.com/ofek/pypinfo CLI.

See https://github.com/python-pillow/Pillow/issues/6941 for some examples.

**Comment by lazka** [comment]
thanks for the pointers!
